phrosty/pipeline.py

```diff
@@ -65,14 +67,28 @@ def __init__( self, imagepath, pointing, sca ):
 
         self.pointing = pointing
 
+        # Intermediate files
+        if self.keep_intermediate:
+            self.skysub_path = None
+            self.detmask_path = None
+            self.input_sci_psf_path = None
+            self.input_templ_psf_path = None
+            self.aligned_sci_img_path = None
+            self.aligned_templ_img_path = None
+            self.aligned_sci_psf_path = None
+            self.aligned_templ_psf_path = None
+            self.crossconv_sci_path = None
+            self.crossconv_templ_path = None
+            self.decorr_kernel_path = None
```

Comment:
I don't understand the logic here.  Why does `keep_intermediate` being `True` set all of these paths for intermediate files to `None`?